recipes/folly/all/conanfile.py

```diff
@@ -91,24 +84,31 @@ def requirements(self):
         # FIXME: Causing compilation issues on clang: self.requires("jemalloc/5.2.1")
         if self.settings.os == "Linux":
             self.requires("libiberty/9.1.0")
-            self.requires("libunwind/1.5.0")
-        if Version(self.version) >= "2020.08.10.00":
+            self.requires("libunwind/1.6.2")
+        if "2020.08.10.00" <= Version(self.version) < "2022.01.31.00":
             self.requires("fmt/7.1.3")
+        if Version(self.version) >= "2022.01.31.00":
+            self.requires("fmt/8.0.1")  # Folly bump fmt to 8.0.1 in v2022.01.31.00
 
     @property
     def _required_boost_components(self):
         return ["context", "filesystem", "program_options", "regex", "system", "thread"]
 
     def validate(self):
-        if self.settings.compiler.get_safe("cppstd"):
-            tools.check_min_cppstd(self, self._minimum_cpp_standard)
-        min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
-        if not min_version:
-            self.output.warn("{} recipe lacks information about the {} compiler support.".format(self.name, self.settings.compiler))
-        else:
-            if Version(self.settings.compiler.version) < min_version:
-                raise ConanInvalidConfiguration("{} requires C++{} support. The current compiler {} {} does not support it.".format(
-                    self.name, self._minimum_cpp_standard, self.settings.compiler, self.settings.compiler.version))
+        compiler_cppstd = self.settings.compiler.get_safe("cppstd")
+        if compiler_cppstd:
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+            )
+        supported_cppstds = supported_cppstd(self)
+        if supported_cppstds and self._min_cppstd not in supported_cppstds:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+                f" Supported cppstds: {supported_cppstds}"
+            )
```

Comment:
This looks out of place 🤔 

```suggestion
            )
```